nntrainer/graph/network_graph.cpp

```diff
@@ -793,9 +787,12 @@ int NetworkGraph::initialize(std::shared_ptr<Manager> manager) {
     lptr->setOutputBuffers(in_out);
 #endif
 
+    if (idx == graph.size() - 1)
```

Comment:
Wonder what this is for, we are skipping the last layer?

Reply:
Yes, for last layer, there is no need to update the input_map anymore. So, it is skipped.
added a comment here.